site/en/blog/private-prefetch-proxy/index.md

```diff
@@ -87,7 +85,16 @@ From past experiments, we know that this feature typically results in less than
 
 The `fraction` field gives you control over the prefetch traffic by specifying how much Private Prefetch Proxy should let through. The value is a float between 0.0 (no prefetch at all) and 1.0 (100% of the prefetch requests get through).
```

Comment:
Up to you, but this sentence is a dupe from above and can probably be simplified by dropping the first sentence.

e.g., "The fraction field is a float between 0.0.."

Reply:
```suggestion
The `fraction` field is a float between 0.0 (no prefetch at all) and 1.0 (100% of the prefetch requests get through).
```